modules/ROOT/pages/manage-client-apps-latest-task.adoc

```diff
@@ -5,12 +5,21 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: grant access, api, contracts
 
-After the user xref:exchange::to-request-access.adoc[requests access] to your API instance from Anypoint Exchange, as the Organization Administrator, you can manually approve or reject requests from API Manager if you have configured the SLA tier for the application for manual approvals.
+After users xref:exchange::to-request-access.adoc[request access] to your API instance from Anypoint Exchange, you can manually approve or reject requests from API Manager. 
+
+== Task Prerequisite
+
+To manually approve or reject requests, you must have:
+
+* The Organization Administrator or Manage Contracts permission
+* Configured the SLA tier for the application 
```

Comment:
this is like a prerequisite right? There wouldn't be nothing to approve/reject if there is no tier 😄

Reply:
Yes, this is a part of the Prerequisite section, right?

Reply:
oh yes, my bad, thanks!